egs/uw3/v1/local/score.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+[ -f ./path.sh ] && . ./path.sh
+
+# begin configuration section.
+cmd=run.pl
+stage=0
+decode_mbr=false
+stats=true
```

Comment:
perhaps you should add `stats` to scoring_opts as well.
Also, if don't wanna do it the way suggested in kaldi_score_cer.sh (I assume because you want to modify the default lmwts, even though I'm not sure if it's worth the efort), please make sure the stage parameter is processed correctly

Reply:
@ChunChiehChang, could you please change this script
to follow the instructions mentioned in the header of `steps/scoring/score_kaldi_cer.sh`?